<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>351</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Хранитель паролей</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>../../Downloads/lock_icon.png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 201)</string>
  </property>
  <property name="iconSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit_search">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(255, 253, 189);
    color: rgb(0, 0, 0);</string>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
        <property name="placeholderText">
         <string>Поиск по меткам</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 253, 189);
    color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Создать новый пароль</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
